include/lug/Window/Window.hpp

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <memory>
+#include <queue>
+#include <lug/Window/Export.hpp>
+
+namespace lug {
+namespace Window {
+namespace priv {
+    class WindowImpl;
+} // namespace priv
+
+enum class LUG_WINDOW_API EventType
+{
```

Comment:
Make the first bracket at the end of the last line.
That applies for all your source codes even in if statement, etc. (**I have seen this a lots of time**)